utils/preferences/Cargo.toml

```
@@ -0,0 +1,29 @@
+# This file is part of ICU4X. For terms of use, please see the file
+# called LICENSE at the top level of the ICU4X source tree
+# (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+[package]
+name = "icu_preferences"
+description = "API for resolving preferences"
+version = "0.0.1"
```

nit: we set unreleased crates to 0.0.0
```suggestion
version = "0.0.0"
```